script.js

```diff
@@ -0,0 +1,91 @@
+const body = document.querySelector('body');
+
+const textArea = document.createElement('textarea');
+textArea.className = 'board';
+body.append(textArea);
+
+const language = document.createElement('div');
+language.className = 'language';
+language.innerHTML = 'Привет!) Чтобы поменять раскладку клавиатуры - нажмите Томатную кнопку и поменяйте раскладку на вашем устройстве)';
+body.append(language);
+
+const keyboard = document.createElement('div');
+keyboard.className = 'keyboard';
+keyboard.id = 'keyboard';
+body.append(keyboard);
+
+const keyboardNumbersRus = [62, 49, 50, 51, 52, 53, 54, 55, 56, 57, 48, 45, 61, 1081,
+  1094, 1091, 1082, 1077, 1085, 1075, 1096, 1097, 1079, 1093, 1098, 1092, 1099, 1074,
+  1072, 1087, 1088, 1086, 1083, 1076, 1078, 1101, 1105, 1103, 1095, 1089, 1084, 1080,
+  1090, 1100, 1073, 1102, 47];
+
+const keyboardNumbersEng = [167, 49, 50, 51, 52, 53, 54, 55, 56, 57, 48, 45, 61, 113,
+  119, 101, 114, 116, 121, 117, 105, 111, 112, 91, 93, 97, 115, 100, 102, 103, 104,
+  106, 107, 108, 59, 92, 122, 120, 99, 118, 98, 110, 109, 44, 46, 47];
+
+let keyboardNumbers = keyboardNumbersEng;
+
+
+function init() {
+  textArea.focus();
+
+  // положили все кнопки в html
+  let out = '';
+  for (let i = 0; i < keyboardNumbers.length; i++) {
+    if (i === 13 || i === 25 || i === 37) {
```

Comment:
What is 13, 25, 37 ??
Why don`t you use keyCodes instead of numbers?

Reply:
It's buttons which i moved on another line)

Reply:
These numbers are charCodes of buttons which i want to move on another line

Reply:
Magic numbers